config/agents/app/rbac/role.yaml

```
@@ -76,3 +76,13 @@ rules:
   - servicebindings/finalizers
   verbs:
   - update
+- apiGroups:
+  - primaza.io
+  resources:
+  - servicecatalogs
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
```

```suggestion
  - patch
  - delete
```

seems to me we need also `delete`.